CMakeLists.txt

```diff
@@ -14,18 +14,24 @@ add_executable(main
   src/Shader.hpp
   src/Shader.cpp
 )
-set_property(TARGET main PROPERTY CXX_STANDARD 11)
-target_compile_options(main PRIVATE -Wall)
+
+add_custom_command(TARGET ${CMAKE_PROJECT_NAME} POST_BUILD
```

Comment:
This is probably going to break if developers are building from the source directory.

I am wondering if we shouldn't instead encode the full path?
I remember this is what I did for the examples of [smk](https://github.com/ArthurSonzogni/smk) here:
https://github.com/ArthurSonzogni/smk/tree/master/examples/assets

Reply:
I'll try that when I get home to be sure.

Reply:
Ran this using CLion and seems to have compiled and executed fine 
<img width="586" alt="image" src="https://user-images.githubusercontent.com/2971844/153105625-969b94c1-d181-42fb-9383-c5e26d31d770.png">

Reply:
And I did also run  it off the command-line with cmake.exe on Windows and still got the build I was expecting.
![image](https://user-images.githubusercontent.com/2971844/153109698-184deffa-9565-479b-9ca1-e7cc800184db.png)

Reply:
And, of course, I originally made this with Visual Studio 2022 using CMake and that built and executed fine also.